build.gradle

```diff
@@ -59,6 +59,7 @@ opensearchplugin {
     classname "${projectPath}.${pathToPlugin}.${pluginClassName}"
     licenseFile rootProject.file('LICENSE')
     noticeFile rootProject.file('NOTICE')
+    extendedPlugins = ['opensearch-knn']
```

Comment:
should we add MLCommons also over here? or this is just for the plugins who we depend during compile time?

Reply:
For ml-commons, we are okay because we just take the client as a dependency.